middleware_url_rewrite.go

```diff
@@ -113,34 +113,10 @@ func (u URLRewriter) Rewrite(thisMeta *tykcommon.URLRewriteMeta, path string, us
 			contextKey := strings.Replace(v[0], "$tyk_meta.", "", 1)
 			log.Debug("Replacing: ", v[0])
 
-			tempVal, ok := thisSessionState.MetaData.(map[string]interface{})[contextKey]
+			tempVal, ok := thisSessionState.MetaData[contextKey]
 			if ok {
-				var nVal string
```

Comment:
Looks like a big chunk of code...

In master we still use valToStr https://github.com/TykTechnologies/tyk/blob/master/mw_url_rewrite.go#L90